src/main.py

```diff
@@ -352,13 +301,21 @@ def setUp(self):
         return 0
 
     def start_server(self):
-        if self.server_thread:
-            del self.server_thread
+        os.environ["KOLIBRI_HTTP_PORT"] = str(self.port)
+
+        if pew.ui.platform == "android":
+            logging.info("Starting kolibri as Android service...")
 
-        logging.info("Preparing to start Kolibri server...")
-        self.server_thread = pew.ui.PEWThread(target=start_django, kwargs={'port': self.port})
-        self.server_thread.daemon = True
-        self.server_thread.start()
+            from platforms.android.utils import start_service
+            start_service("kolibri", dict(os.environ))
+        else:
+            if self.server_thread:
+                del self.server_thread
```

Comment:
Yes, good point, I was treating this as a next step, but it makes sense to go ahead and do this now in order to simplify the startup code as well.

Reply:
@dylanmccall On second thought, going to handle this as a subsequent PR after merging this one, so we can finish up the review on this, since it's become quite meaty. :)

Reply:
Issue filed here: https://github.com/learningequality/kolibri-installer-mac/issues/73